src/commons/api/history.rs

```diff
@@ -118,19 +58,20 @@ impl CommandHistory {
 
 impl fmt::Display for CommandHistory {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        writeln!(f, "time::command::key::success")?;
+        writeln!(f, "time::command::sequence::success")?;
```

Comment:
You say `sequence` here but below output a value called `version` - is `sequence` the right term to use here?

Reply:
Missed that one. I will do a search for 'sequence' and replace it with 'version'.